Fixes #10070 - Adds exception for `div` inside `a` (#13165)
mdn/content: files/en-us/web/html/inline_elements/index.md

In brief, here are the basic conceptual differences between inline and block-level elements:

- Content model

>   - : Generally, inline elements may contain only data and other inline elements. An exception is the inline `a` element which may contain block level elements such as `div`.

Reviewer: Could we also add a note that links that wrap multiple lines of block-level content makes for a poor-to-unusable experience for some assistive technology? It's one of those things where just because you technically can doesn't mean you should in the practical.

In brief, here are the basic conceptual differences between inline and block-level elements:

- Content model
  - : Generally, inline elements may contain only data and other inline elements. An exception is the inline `a` element which may contain block level elements such as `div`.

>     > **Note:** Links that wrap multiple lines of block-level content make for a poor-to-unusable experience for some assistive technology. It is one of those things where just because you technically can does not mean you should in the practical.

Reviewer 2: ```suggestion
    > **Note:** Links that wrap multiple lines of block-level content make for a poor-to-unusable experience for some assistive technologies and should be avoided.
```
